projects/firefox/default.options

```diff
@@ -0,0 +1,4 @@
+[asan]
```

Comment:
`default` should be replaced with the fuzz target binary name

Reply:
and you also need to update the Dockerfile to copy it into the build container

Reply:
this logic should be documented, it's similar to the `<fuzz_target>_seed_corpus.zip` feature

Reply:
>
>
> `default` should be replaced with the fuzz target binary name

From your previous comment I was hoping that `default.options` was an undocumented catch all. OK will create individual .options files

> 
> 
> and you also need to update the Dockerfile to copy it into the build container

That is already the [case
](https://github.com/google/oss-fuzz/blob/master/projects/firefox/Dockerfile#L28) or am I missing something?

Also the CI seems to be failing. It works locally and the log are unavailable `We are currently unable to download the log. Please try again later.` Can you see what's wrong?

Reply:
https://github.com/google/oss-fuzz/pull/4258/checks?check_run_id=951560825
You should be able to expand the Run project tests part

```
2020-08-06T01:32:25.3878727Z Status: Downloaded newer image for gcr.io/oss-fuzz-base/base-runner:latest
2020-08-06T01:32:32.6809496Z INFO: performing bad build checks for /tmp/not-out/ImageGIF.
2020-08-06T01:32:32.7334787Z INFO: performing bad build checks for /tmp/not-out/SdpParser.
2020-08-06T01:32:45.1369144Z INFO: performing bad build checks for /tmp/not-out/MediaOgg.
2020-08-06T01:32:45.5275667Z INFO: performing bad build checks for /tmp/not-out/ContentParentIPC.
2020-08-06T01:32:53.5612111Z INFO: performing bad build checks for /tmp/not-out/FeaturePolicyParser.
2020-08-06T01:32:53.5778284Z INFO: performing bad build checks for /tmp/not-out/MediaADTS.
2020-08-06T01:33:01.6193280Z INFO: performing bad build checks for /tmp/not-out/MediaFlac.
2020-08-06T01:33:01.6307764Z INFO: performing bad build checks for /tmp/not-out/CompositorManagerParentIPC.
2020-08-06T01:33:09.6650763Z INFO: performing bad build checks for /tmp/not-out/MediaMP3.
2020-08-06T01:33:09.6889403Z INFO: performing bad build checks for /tmp/not-out/ImageICO.
2020-08-06T01:33:17.7394867Z INFO: performing bad build checks for /tmp/not-out/ImageBMP.
2020-08-06T01:33:17.7491051Z INFO: performing bad build checks for /tmp/not-out/ContentSecurityPolicyParser.
2020-08-06T01:33:25.7891460Z INFO: performing bad build checks for /tmp/not-out/StunParser.
2020-08-06T01:33:25.8063152Z INFO: performing bad build checks for /tmp/not-out/MediaWebM.
2020-08-06T01:33:39.1904022Z Broken fuzz targets (14):
2020-08-06T01:33:39.1914942Z CompositorManagerParentIPC:
2020-08-06T01:33:39.1928294Z BAD BUILD: /tmp/not-out/CompositorManagerParentIPC seems to have either startup crash or exit:
2020-08-06T01:33:39.1928709Z *** You are running in headless mode.
2020-08-06T01:33:39.1928842Z Running Fuzzer tests...
2020-08-06T01:33:39.1929147Z too much recursion
2020-08-06T01:33:39.1929325Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.1929482Z =================================================================
2020-08-06T01:33:39.1929683Z ==448==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f6ccaae5ffb bp 0x7ffc3e7d98d0 sp 0x7ffc3e7d98d0 T0)
2020-08-06T01:33:39.1931669Z ==448==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.1932395Z ==448==Hint: address points to the zero page.
2020-08-06T01:33:39.1932960Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.1933131Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.1933581Z     #0 0x7f6ccaae5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.1934504Z     #1 0x7f6ccca66f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.1935158Z     #2 0x7f6ccca67f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.1935776Z     #3 0x7f6cccae9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.1936187Z     #4 0x7f6ccad545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2302802Z     #5 0x7f6cd4f4856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2303847Z     #6 0x7f6cd4f49ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2304785Z     #7 0x7f6cd4f4826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2305679Z     #8 0x7f6cd4e85835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2306589Z     #9 0x7f6cd4e8de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2307521Z     #10 0x7f6cd4e8e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2308095Z     #11 0x562f96be9ef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2308612Z     #12 0x562f96be95e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2309143Z     #13 0x7f6ceeab683f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2309661Z     #14 0x562f96b3e028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2309799Z 
2020-08-06T01:33:39.2310605Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2310846Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2311435Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2311662Z ==448==ABORTING
2020-08-06T01:33:39.2311848Z ContentParentIPC:
2020-08-06T01:33:39.2312664Z BAD BUILD: /tmp/not-out/ContentParentIPC seems to have either startup crash or exit:
2020-08-06T01:33:39.2312892Z *** You are running in headless mode.
2020-08-06T01:33:39.2313068Z Running Fuzzer tests...
2020-08-06T01:33:39.2313253Z too much recursion
2020-08-06T01:33:39.2313443Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2313645Z =================================================================
2020-08-06T01:33:39.2313908Z ==229==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f26531e5ffb bp 0x7ffc2b89d030 sp 0x7ffc2b89d030 T0)
2020-08-06T01:33:39.2317405Z ==229==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2319528Z ==229==Hint: address points to the zero page.
2020-08-06T01:33:39.2320264Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2320476Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2321096Z     #0 0x7f26531e5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2322196Z     #1 0x7f2655166f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2323306Z     #2 0x7f2655167f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2324394Z     #3 0x7f26551e9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2325181Z     #4 0x7f26534545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2326230Z     #5 0x7f265d64856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2327162Z     #6 0x7f265d649ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2328066Z     #7 0x7f265d64826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2328927Z     #8 0x7f265d585835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2329834Z     #9 0x7f265d58de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2330773Z     #10 0x7f265d58e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2331359Z     #11 0x5564d4a40ef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2331882Z     #12 0x5564d4a405e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2332418Z     #13 0x7f26771ea83f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2332939Z     #14 0x5564d4995028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2333071Z 
2020-08-06T01:33:39.2333871Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2334108Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2334692Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2334920Z ==229==ABORTING
2020-08-06T01:33:39.2335103Z ContentSecurityPolicyParser:
2020-08-06T01:33:39.2335647Z BAD BUILD: /tmp/not-out/ContentSecurityPolicyParser seems to have either startup crash or exit:
2020-08-06T01:33:39.2335871Z *** You are running in headless mode.
2020-08-06T01:33:39.2336066Z Running Fuzzer tests...
2020-08-06T01:33:39.2336236Z too much recursion
2020-08-06T01:33:39.2336424Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2336630Z =================================================================
2020-08-06T01:33:39.2336876Z ==676==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7fa2663e5ffb bp 0x7ffc05701bf0 sp 0x7ffc05701bf0 T0)
2020-08-06T01:33:39.2337115Z ==676==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2337323Z ==676==Hint: address points to the zero page.
2020-08-06T01:33:39.2337738Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2337959Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2338529Z     #0 0x7fa2663e5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2339538Z     #1 0x7fa268366f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2340402Z     #2 0x7fa268367f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2341251Z     #3 0x7fa2683e9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2341814Z     #4 0x7fa2666545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2342809Z     #5 0x7fa27084856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2343732Z     #6 0x7fa270849ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2344712Z     #7 0x7fa27084826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2345597Z     #8 0x7fa270785835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2346499Z     #9 0x7fa27078de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2347422Z     #10 0x7fa27078e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2347990Z     #11 0x55c2bf8a7ef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2348509Z     #12 0x55c2bf8a75e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2349036Z     #13 0x7fa28a42b83f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2349552Z     #14 0x55c2bf7fc028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2349707Z 
2020-08-06T01:33:39.2350492Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2350730Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2351311Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2351539Z ==676==ABORTING
2020-08-06T01:33:39.2351722Z FeaturePolicyParser:
2020-08-06T01:33:39.2352227Z BAD BUILD: /tmp/not-out/FeaturePolicyParser seems to have either startup crash or exit:
2020-08-06T01:33:39.2352449Z *** You are running in headless mode.
2020-08-06T01:33:39.2352637Z Running Fuzzer tests...
2020-08-06T01:33:39.2352805Z too much recursion
2020-08-06T01:33:39.2352992Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2353197Z =================================================================
2020-08-06T01:33:39.2353451Z ==334==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f58005e5ffb bp 0x7ffd5639cbb0 sp 0x7ffd5639cbb0 T0)
2020-08-06T01:33:39.2353693Z ==334==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2353902Z ==334==Hint: address points to the zero page.
2020-08-06T01:33:39.2354315Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2354536Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2355122Z     #0 0x7f58005e5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2355958Z     #1 0x7f5802566f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2357052Z     #2 0x7f5802567f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2357904Z     #3 0x7f58025e9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2358473Z     #4 0x7f58008545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2359652Z     #5 0x7f580aa4856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2360135Z     #6 0x7f580aa49ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2360606Z     #7 0x7f580aa4826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2361052Z     #8 0x7f580a985835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2361530Z     #9 0x7f580a98de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2362046Z     #10 0x7f580a98e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2362362Z     #11 0x55c2ad1daef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2362638Z     #12 0x55c2ad1da5e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2362917Z     #13 0x7f582464883f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2363194Z     #14 0x55c2ad12f028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2363275Z 
2020-08-06T01:33:39.2363892Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2364036Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2364385Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2364524Z ==334==ABORTING
2020-08-06T01:33:39.2364635Z ImageBMP:
2020-08-06T01:33:39.2364938Z BAD BUILD: /tmp/not-out/ImageBMP seems to have either startup crash or exit:
2020-08-06T01:33:39.2365071Z *** You are running in headless mode.
2020-08-06T01:33:39.2365191Z Running Fuzzer tests...
2020-08-06T01:33:39.2365304Z too much recursion
2020-08-06T01:33:39.2365409Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2365531Z =================================================================
2020-08-06T01:33:39.2365681Z ==673==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f5682de5ffb bp 0x7ffeede0ac30 sp 0x7ffeede0ac30 T0)
2020-08-06T01:33:39.2365825Z ==673==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2365951Z ==673==Hint: address points to the zero page.
2020-08-06T01:33:39.2366199Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2366336Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2366696Z     #0 0x7f5682de5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2367221Z     #1 0x7f5684d66f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2367743Z     #2 0x7f5684d67f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2368257Z     #3 0x7f5684de9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2368581Z     #4 0x7f56830545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2369177Z     #5 0x7f568d24856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2369737Z     #6 0x7f568d249ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2370282Z     #7 0x7f568d24826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2370864Z     #8 0x7f568d185835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2371402Z     #9 0x7f568d18de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2371955Z     #10 0x7f568d18e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2372299Z     #11 0x5587f9524ef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2372595Z     #12 0x5587f95245e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2372915Z     #13 0x7f56a6e1e83f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2373229Z     #14 0x5587f9479028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2373323Z 
2020-08-06T01:33:39.2373847Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2373999Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2374340Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2374479Z ==673==ABORTING
2020-08-06T01:33:39.2374594Z ImageGIF:
2020-08-06T01:33:39.2374886Z BAD BUILD: /tmp/not-out/ImageGIF seems to have either startup crash or exit:
2020-08-06T01:33:39.2375023Z *** You are running in headless mode.
2020-08-06T01:33:39.2375142Z Running Fuzzer tests...
2020-08-06T01:33:39.2375259Z too much recursion
2020-08-06T01:33:39.2375376Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2375487Z =================================================================
2020-08-06T01:33:39.2375639Z ==107==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f7474ce5ffb bp 0x7ffce7ba2430 sp 0x7ffce7ba2430 T0)
2020-08-06T01:33:39.2375794Z ==107==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2375922Z ==107==Hint: address points to the zero page.
2020-08-06T01:33:39.2376287Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2376410Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2376936Z     #0 0x7f7474ce5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2377447Z     #1 0x7f7476c66f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2377960Z     #2 0x7f7476c67f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2378473Z     #3 0x7f7476ce9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2378812Z     #4 0x7f7474f545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2379421Z     #5 0x7f747f14856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2379973Z     #6 0x7f747f149ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2380510Z     #7 0x7f747f14826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2381018Z     #8 0x7f747f085835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2381744Z     #9 0x7f747f08de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2382393Z     #10 0x7f747f08e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2382837Z     #11 0x562b1fcd8ef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2383282Z     #12 0x562b1fcd85e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2383586Z     #13 0x7f7498ca183f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2383901Z     #14 0x562b1fc2d028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2383992Z 
2020-08-06T01:33:39.2384473Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2384621Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2384970Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2385097Z ==107==ABORTING
2020-08-06T01:33:39.2385210Z ImageICO:
2020-08-06T01:33:39.2385550Z BAD BUILD: /tmp/not-out/ImageICO seems to have either startup crash or exit:
2020-08-06T01:33:39.2385690Z *** You are running in headless mode.
2020-08-06T01:33:39.2385809Z Running Fuzzer tests...
2020-08-06T01:33:39.2385924Z too much recursion
2020-08-06T01:33:39.2386040Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2386152Z =================================================================
2020-08-06T01:33:39.2386302Z ==562==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f36f4fe5ffb bp 0x7ffe233602d0 sp 0x7ffe233602d0 T0)
2020-08-06T01:33:39.2386448Z ==562==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2386577Z ==562==Hint: address points to the zero page.
2020-08-06T01:33:39.2386832Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2386970Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2387324Z     #0 0x7f36f4fe5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2387844Z     #1 0x7f36f6f66f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2388558Z     #2 0x7f36f6f67f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2389110Z     #3 0x7f36f6fe9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2389571Z     #4 0x7f36f52545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2390493Z     #5 0x7f36ff44856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2391037Z     #6 0x7f36ff449ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2391682Z     #7 0x7f36ff44826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2392138Z     #8 0x7f36ff385835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2392607Z     #9 0x7f36ff38de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2393091Z     #10 0x7f36ff38e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2393392Z     #11 0x55aa253aaef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2393666Z     #12 0x55aa253aa5e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2393947Z     #13 0x7f371903783f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2394210Z     #14 0x55aa252ff028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2394293Z 
2020-08-06T01:33:39.2394767Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2394899Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2395205Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2395314Z ==562==ABORTING
2020-08-06T01:33:39.2395414Z MediaADTS:
2020-08-06T01:33:39.2395671Z BAD BUILD: /tmp/not-out/MediaADTS seems to have either startup crash or exit:
2020-08-06T01:33:39.2395972Z *** You are running in headless mode.
2020-08-06T01:33:39.2396264Z Running Fuzzer tests...
2020-08-06T01:33:39.2396380Z too much recursion
2020-08-06T01:33:39.2396497Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2396622Z =================================================================
2020-08-06T01:33:39.2396761Z ==331==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f81d64e5ffb bp 0x7ffe1bda6d30 sp 0x7ffe1bda6d30 T0)
2020-08-06T01:33:39.2396988Z ==331==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2397122Z ==331==Hint: address points to the zero page.
2020-08-06T01:33:39.2397395Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2397534Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2397888Z     #0 0x7f81d64e5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2398406Z     #1 0x7f81d8466f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2398924Z     #2 0x7f81d8467f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2399436Z     #3 0x7f81d84e9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2399778Z     #4 0x7f81d67545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2400389Z     #5 0x7f81e094856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2400940Z     #6 0x7f81e0949ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2401480Z     #7 0x7f81e094826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2401988Z     #8 0x7f81e0885835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2402524Z     #9 0x7f81e088de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2403082Z     #10 0x7f81e088e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2403436Z     #11 0x55bd83800ef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2403754Z     #12 0x55bd838005e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2404079Z     #13 0x7f81fa54983f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2404379Z     #14 0x55bd83755028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2404470Z 
2020-08-06T01:33:39.2404949Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2405094Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2405442Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2405581Z ==331==ABORTING
2020-08-06T01:33:39.2405679Z MediaFlac:
2020-08-06T01:33:39.2405973Z BAD BUILD: /tmp/not-out/MediaFlac seems to have either startup crash or exit:
2020-08-06T01:33:39.2406196Z *** You are running in headless mode.
2020-08-06T01:33:39.2406318Z Running Fuzzer tests...
2020-08-06T01:33:39.2406431Z too much recursion
2020-08-06T01:33:39.2406549Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2406673Z =================================================================
2020-08-06T01:33:39.2406828Z ==445==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7fcb513e5ffb bp 0x7ffd8773b190 sp 0x7ffd8773b190 T0)
2020-08-06T01:33:39.2406958Z ==445==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2407089Z ==445==Hint: address points to the zero page.
2020-08-06T01:33:39.2407349Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2407488Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2407840Z     #0 0x7fcb513e5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2408400Z     #1 0x7fcb53366f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2408935Z     #2 0x7fcb53367f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2409444Z     #3 0x7fcb533e9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2409778Z     #4 0x7fcb516545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2410374Z     #5 0x7fcb5b84856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2410915Z     #6 0x7fcb5b849ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2411448Z     #7 0x7fcb5b84826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2411973Z     #8 0x7fcb5b785835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2412511Z     #9 0x7fcb5b78de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2413061Z     #10 0x7fcb5b78e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2413405Z     #11 0x56290524def4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2413725Z     #12 0x56290524d5e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2414046Z     #13 0x7fcb753ab83f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2414361Z     #14 0x5629051a2028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2414436Z 
2020-08-06T01:33:39.2414913Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2415061Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2415411Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2415552Z ==445==ABORTING
2020-08-06T01:33:39.2415652Z MediaMP3:
2020-08-06T01:33:39.2415947Z BAD BUILD: /tmp/not-out/MediaMP3 seems to have either startup crash or exit:
2020-08-06T01:33:39.2416081Z *** You are running in headless mode.
2020-08-06T01:33:39.2416200Z Running Fuzzer tests...
2020-08-06T01:33:39.2416312Z too much recursion
2020-08-06T01:33:39.2416427Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2416552Z =================================================================
2020-08-06T01:33:39.2416704Z ==559==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7fcf310e5ffb bp 0x7fff1b4f2450 sp 0x7fff1b4f2450 T0)
2020-08-06T01:33:39.2416837Z ==559==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2417017Z ==559==Hint: address points to the zero page.
2020-08-06T01:33:39.2417276Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2417411Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2417764Z     #0 0x7fcf310e5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2418274Z     #1 0x7fcf33066f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2418785Z     #2 0x7fcf33067f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2419289Z     #3 0x7fcf330e9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2419627Z     #4 0x7fcf313545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2420268Z     #5 0x7fcf3b54856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2420835Z     #6 0x7fcf3b549ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2421367Z     #7 0x7fcf3b54826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2421878Z     #8 0x7fcf3b485835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2422410Z     #9 0x7fcf3b48de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2422962Z     #10 0x7fcf3b48e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2423316Z     #11 0x562d132b9ef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2423630Z     #12 0x562d132b95e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2423949Z     #13 0x7fcf5509b83f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2424261Z     #14 0x562d1320e028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2424336Z 
2020-08-06T01:33:39.2424808Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2424955Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2425514Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2425623Z ==559==ABORTING
2020-08-06T01:33:39.2425711Z MediaOgg:
2020-08-06T01:33:39.2425919Z BAD BUILD: /tmp/not-out/MediaOgg seems to have either startup crash or exit:
2020-08-06T01:33:39.2426023Z *** You are running in headless mode.
2020-08-06T01:33:39.2426122Z Running Fuzzer tests...
2020-08-06T01:33:39.2426212Z too much recursion
2020-08-06T01:33:39.2426302Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2426399Z =================================================================
2020-08-06T01:33:39.2426516Z ==210==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f02713e5ffb bp 0x7ffe7cfa8110 sp 0x7ffe7cfa8110 T0)
2020-08-06T01:33:39.2426625Z ==210==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2426711Z ==210==Hint: address points to the zero page.
2020-08-06T01:33:39.2426901Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2427180Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2427693Z     #0 0x7f02713e5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2428253Z     #1 0x7f0273366f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2428828Z     #2 0x7f0273367f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2429331Z     #3 0x7f02733e9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2429664Z     #4 0x7f02716545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2430264Z     #5 0x7f027b84856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2430808Z     #6 0x7f027b849ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2431344Z     #7 0x7f027b84826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2432008Z     #8 0x7f027b785835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2432563Z     #9 0x7f027b78de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2433114Z     #10 0x7f027b78e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2433458Z     #11 0x558aafacdef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2433773Z     #12 0x558aafacd5e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2434093Z     #13 0x7f029537883f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2434406Z     #14 0x558aafa22028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2434482Z 
2020-08-06T01:33:39.2434963Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2435117Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2435467Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2435608Z ==210==ABORTING
2020-08-06T01:33:39.2435722Z MediaWebM:
2020-08-06T01:33:39.2436001Z BAD BUILD: /tmp/not-out/MediaWebM seems to have either startup crash or exit:
2020-08-06T01:33:39.2436298Z *** You are running in headless mode.
2020-08-06T01:33:39.2436593Z Running Fuzzer tests...
2020-08-06T01:33:39.2436715Z too much recursion
2020-08-06T01:33:39.2436842Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2436978Z =================================================================
2020-08-06T01:33:39.2437139Z ==790==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f557e8e5ffb bp 0x7ffcf241ab70 sp 0x7ffcf241ab70 T0)
2020-08-06T01:33:39.2437291Z ==790==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2437422Z ==790==Hint: address points to the zero page.
2020-08-06T01:33:39.2437705Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2437853Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2438235Z     #0 0x7f557e8e5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2438786Z     #1 0x7f5580866f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2439440Z     #2 0x7f5580867f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2440053Z     #3 0x7f55808e9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2440350Z     #4 0x7f557eb545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2440876Z     #5 0x7f5588d4856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2441421Z     #6 0x7f5588d49ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2441892Z     #7 0x7f5588d4826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2442339Z     #8 0x7f5588c85835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2442810Z     #9 0x7f5588c8de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2443292Z     #10 0x7f5588c8e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2443847Z     #11 0x55cbb7574ef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2444176Z     #12 0x55cbb75745e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2444496Z     #13 0x7f55a296e83f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2444808Z     #14 0x55cbb74c9028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2444883Z 
2020-08-06T01:33:39.2445361Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2445508Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2445856Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2445997Z ==790==ABORTING
2020-08-06T01:33:39.2446109Z SdpParser:
2020-08-06T01:33:39.2446404Z BAD BUILD: /tmp/not-out/SdpParser seems to have either startup crash or exit:
2020-08-06T01:33:39.2446526Z *** You are running in headless mode.
2020-08-06T01:33:39.2446653Z Running Fuzzer tests...
2020-08-06T01:33:39.2446773Z too much recursion
2020-08-06T01:33:39.2446887Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2447013Z =================================================================
2020-08-06T01:33:39.2447166Z ==99==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f9fbbfe5ffb bp 0x7ffdfffbe130 sp 0x7ffdfffbe130 T0)
2020-08-06T01:33:39.2447309Z ==99==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2447439Z ==99==Hint: address points to the zero page.
2020-08-06T01:33:39.2447675Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2447816Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2448169Z     #0 0x7f9fbbfe5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2448676Z     #1 0x7f9fbdf66f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2449197Z     #2 0x7f9fbdf67f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2449706Z     #3 0x7f9fbdfe9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2450043Z     #4 0x7f9fbc2545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2450643Z     #5 0x7f9fc644856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2451188Z     #6 0x7f9fc6449ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2451724Z     #7 0x7f9fc644826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2452239Z     #8 0x7f9fc6385835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2453331Z     #9 0x7f9fc638de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2453928Z     #10 0x7f9fc638e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2454299Z     #11 0x56133d8fbef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2454635Z     #12 0x56133d8fb5e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2454978Z     #13 0x7f9fdffe583f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2455315Z     #14 0x56133d850028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2455396Z 
2020-08-06T01:33:39.2455907Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2456166Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2456555Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2456703Z ==99==ABORTING
2020-08-06T01:33:39.2456822Z StunParser:
2020-08-06T01:33:39.2457139Z BAD BUILD: /tmp/not-out/StunParser seems to have either startup crash or exit:
2020-08-06T01:33:39.2457269Z *** You are running in headless mode.
2020-08-06T01:33:39.2457399Z Running Fuzzer tests...
2020-08-06T01:33:39.2457524Z too much recursion
2020-08-06T01:33:39.2457653Z AddressSanitizer:DEADLYSIGNAL
2020-08-06T01:33:39.2457786Z =================================================================
2020-08-06T01:33:39.2457951Z ==787==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7f6a1f2e5ffb bp 0x7ffea2e1f890 sp 0x7ffea2e1f890 T0)
2020-08-06T01:33:39.2458106Z ==787==The signal is caused by a WRITE memory access.
2020-08-06T01:33:39.2458249Z ==787==Hint: address points to the zero page.
2020-08-06T01:33:39.2458511Z SCARINESS: 10 (null-deref)
2020-08-06T01:33:39.2458660Z error: address range table at offset 0xd50 has an invalid tuple (length = 0) at offset 0xd60
2020-08-06T01:33:39.2459041Z     #0 0x7f6a1f2e5ffa in NS_ABORT_OOM(unsigned long) /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3
2020-08-06T01:33:39.2459589Z     #1 0x7f6a21266f2e in XPCJSContext::Initialize() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1297:7
2020-08-06T01:33:39.2460137Z     #2 0x7f6a21267f02 in XPCJSContext::NewXPCJSContext() /src/mozilla-central/js/xpconnect/src/XPCJSContext.cpp:1330:23
2020-08-06T01:33:39.2460681Z     #3 0x7f6a212e9de8 in nsXPConnect::InitJSContext() /src/mozilla-central/js/xpconnect/src/nsXPConnect.cpp:80:25
2020-08-06T01:33:39.2461039Z     #4 0x7f6a1f5545bc in NS_InitXPCOM /src/mozilla-central/xpcom/build/XPCOMInit.cpp:491:5
2020-08-06T01:33:39.2461680Z     #5 0x7f6a2974856e in (anonymous namespace)::ScopedXPCOM::ScopedXPCOM(char const*, nsIDirectoryServiceProvider*) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerTestHarness.h:69:19
2020-08-06T01:33:39.2462275Z     #6 0x7f6a29749ebe in mozilla::_InitFuzzer::InitXPCOM() /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:23:41
2020-08-06T01:33:39.2462852Z     #7 0x7f6a2974826d in mozilla::FuzzerRunner::Run(int*, char***) /src/mozilla-central/tools/fuzzing/interface/harness/FuzzerRunner.cpp:45:17
2020-08-06T01:33:39.2463402Z     #8 0x7f6a29685835 in XREMain::XRE_mainStartup(bool*) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:3852:35
2020-08-06T01:33:39.2463978Z     #9 0x7f6a2968de33 in XREMain::XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:4945:12
2020-08-06T01:33:39.2464567Z     #10 0x7f6a2968e433 in XRE_main(int, char**, mozilla::BootstrapConfig const&) /src/mozilla-central/toolkit/xre/nsAppRunner.cpp:5012:21
2020-08-06T01:33:39.2464934Z     #11 0x564b8ba60ef4 in do_main(int, char**, char**) (/tmp/not-out/firefox/firefox+0x1d7ef4)
2020-08-06T01:33:39.2465340Z     #12 0x564b8ba605e9 in main (/tmp/not-out/firefox/firefox+0x1d75e9)
2020-08-06T01:33:39.2465789Z     #13 0x7f6a4336683f in __libc_start_main (/tmp/not-out/lib/libc.so.6+0x2083f)
2020-08-06T01:33:39.2466100Z     #14 0x564b8b9b5028 in _start (/tmp/not-out/firefox/firefox+0x12c028)
2020-08-06T01:33:39.2466174Z 
2020-08-06T01:33:39.2466650Z DEDUP_TOKEN: NS_ABORT_OOM(unsigned long)--XPCJSContext::Initialize()--XPCJSContext::NewXPCJSContext()
2020-08-06T01:33:39.2466795Z AddressSanitizer can not provide additional info.
2020-08-06T01:33:39.2467142Z SUMMARY: AddressSanitizer: SEGV /src/mozilla-central/xpcom/base/nsDebugImpl.cpp:620:3 in NS_ABORT_OOM(unsigned long)
2020-08-06T01:33:39.2467282Z ==787==ABORTING
2020-08-06T01:33:39.2467411Z ERROR: 100% of fuzz targets seem to be broken. See the list above for a detailed information.
2020-08-06T01:33:53.4983049Z Running: docker run --rm --privileged -e FUZZING_ENGINE=libfuzzer -e SANITIZER=address -e ARCHITECTURE=x86_64 -v /home/runner/work/oss-fuzz/oss-fuzz/build/out/firefox:/out -t gcr.io/oss-fuzz-base/base-runner test_all
2020-08-06T01:33:53.5037875Z Check build failed.
2020-08-06T01:33:53.5038259Z Building project firefox
2020-08-06T01:33:53.5038992Z Running command: python /home/runner/work/oss-fuzz/oss-fuzz/infra/helper.py build_fuzzers firefox --engine libfuzzer --sanitizer address --architecture x86_64
2020-08-06T01:33:53.5039558Z Running command: python /home/runner/work/oss-fuzz/oss-fuzz/infra/helper.py check_build firefox --engine libfuzzer --sanitizer address --architecture x86_64
2020-08-06T01:33:53.5039821Z Failed projects: firefox
2020-08-06T01:33:53.5145190Z ##[error]Process completed with exit code 1.
2020-08-06T01:33:53.5828177Z Post job cleanup.
2020-08-06T01:33:54.1090280Z [command]/usr/bin/git version
2020-08-06T01:33:54.1495141Z git version 2.27.0
2020-08-06T01:33:54.1550132Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-08-06T01:33:54.1606195Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-08-06T01:33:54.2140257Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-08-06T01:33:54.2178864Z http.https://github.com/.extraheader
2020-08-06T01:33:54.2187128Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2020-08-06T01:33:54.2226409Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-08-06T01:33:54.2560860Z Cleaning up orphan processes
```

Reply:
You should be able to reproduce this easily with:
https://google.github.io/oss-fuzz/advanced-topics/reproducing/#reproducing-build-failures
Never need to rely on CI, local rocks :)

Reply:
OK, I found the problem. It seems that `<target>.options` is not being loaded.

For example I have a `ImageBMP.options` which contains:
```
[asan]
max_allocation_size_mb=2049
```

If I `getenv("ASAN_OPTIONS")` and print it in `target.c` I run `python infra/helper.py reproduce firefox ImageBMP testcase.bmp` and get:
```
alloc_dealloc_mismatch=0:allocator_may_return_null=1:allocator_release_to_os_interval_ms=500:check_malloc_usable_size=0:detect_container_overflow=1:detect_odr_violation=0:detect_leaks=1:detect_stack_use_after_return=1:fast_unwind_on_fatal=0:handle_abort=1:handle_segv=1:handle_sigill=1:max_uar_stack_size_log=16:print_scariness=1:quarantine_size_mb=10:strict_memcmp=1:strip_path_prefix=/workspace/:symbolize=1:use_sigaltstack=1:dedup_token_length=3
```

I don't see `max_allocation_size_mb=2049`. I've also tried adding `detect_leaks=0` to the `ImageBMP.options` as a test to see if that would affect the environment and it does not.

Any tips?

Reply:
@tysmith I'm sorry, we're hitting all the edge cases here. I don't think that OSS-Fuzz scripts are loading sanitizer options from the .options files

as for the `default.options` file, the wasmtime project is copying it for each fuzz targets like this: https://github.com/google/oss-fuzz/blob/3370f8f7e5d93a2f3f3bc95643dcdb350b257d22/projects/wasmtime/build.sh#L43

Reply:
> @tysmith I'm sorry, we're hitting all the edge cases here. I don't think that OSS-Fuzz scripts are loading sanitizer options from the .options files

even if we don't document that, we still need to update the scripts in OSS-Fuzz to be compatible, sorry again for the trouble

Reply:
@Dor1s Do you plan to update the OSS-Fuzz scripts soon? or should I just revert back to doing the ASAN_OPTIONS modification in `target.c` for now?

Reply:
Let me try adding support for that today

Reply:
Uploaded #4274

Reply:
@tysmith should be fine now, could you please rebase and update the PR?

Reply:
I tested locally this morning after a pull and rebase. I removed the modification of ASAN_OPTIONS from target.c and add a `.options` file for each target. I now see the error `NS_ABORT_OOM` from before which indicate that `ASAN_OPTIONS=detect_stack_use_after_return=0` is not set.

For example the `ImageBMP.options` file not contains:
```
[asan]
detect_leaks=0
detect_stack_use_after_return=0
max_allocation_size_mb=4097

[libfuzzer]
close_fd_mask = 3
```

Reply:
oh, we don't use `run_fuzzer` when testing libFuzzer fuzz targets for a startup crash, that's why my change didn't have effect :( however we do use `run_fuzzer` for AFL, perhaps that's what confused me. My apologies once again, let me see what's the best way to make it work, and I'll verify myself using the `.options` contents from your comment and removing env modification from the fuzz target.

Reply:
alright, with the following patch:

```diff
$ git diff
diff --git a/infra/base-images/base-runner/bad_build_check b/infra/base-images/base-runner/bad_build_check
index 4ec23f68..24856a14 100755
--- a/infra/base-images/base-runner/bad_build_check
+++ b/infra/base-images/base-runner/bad_build_check
@@ -130,7 +130,7 @@ function check_startup_crash {
   local CHECK_PASSED=0
 
   if [[ "$FUZZING_ENGINE" = libfuzzer ]]; then
-    $FUZZER -seed=1337 -runs=$MIN_NUMBER_OF_RUNS &>$FUZZER_OUTPUT
+    SKIP_SEED_CORPUS=0 run_fuzzer $FUZZER_NAME -seed=1337 -runs=$MIN_NUMBER_OF_RUNS &>$FUZZER_OUTPUT
     CHECK_PASSED=$(egrep "Done $MIN_NUMBER_OF_RUNS runs" -c $FUZZER_OUTPUT)
   elif [[ "$FUZZING_ENGINE" = afl ]]; then
     # TODO(https://github.com/google/oss-fuzz/issues/2470): Dont use
diff --git a/projects/firefox/ImageBMP.options b/projects/firefox/ImageBMP.options
index 329a6e27..ed3fa273 100644
--- a/projects/firefox/ImageBMP.options
+++ b/projects/firefox/ImageBMP.options
@@ -1,2 +1,7 @@
+[asan]
+detect_leaks=0
+detect_stack_use_after_return=0
+max_allocation_size_mb=4097
+
 [libfuzzer]
 close_fd_mask = 3
diff --git a/projects/firefox/target.c b/projects/firefox/target.c
index 3adfd98e..ed09a1f6 100644
--- a/projects/firefox/target.c
+++ b/projects/firefox/target.c
@@ -61,7 +61,7 @@ int main(int argc, char* argv[]) {
   setenv("MOZ_IPC_MESSAGE_FUZZ_BLACKLIST", blacklist_path, 1);
 
   // Temporary (or permanent?) work-arounds for fuzzing interface bugs.
-  char* options = getenv("ASAN_OPTIONS");
+  char* options = 0; // getenv("ASAN_OPTIONS");
   const char max_alloc_size_opt[] = ":max_allocation_size_mb=2049";
   if (options) {
     size_t optlen = strlen(options);
```

ImageBMP fuzz target passes the build check. I'll land the `run_fuzzer` change.

Reply:
Great, thank you!

Reply:
I landed my fix, but our base-images build is broken, so base-runner doesn't get updated until we fix: https://github.com/google/oss-fuzz/issues/4270

Reply:
alright, NOW it should finally work :)

you can do `python infra/helper.py pull_images` locally, or just update the PR to have `.options` file as discussed above

Reply:
I did a `python infra/helper.py pull_image` and I noticed that `python infra/helper.py reproduce` does not seem to work with `.options` files. This is also going to bite me if I make the switch.

Reply:
Wow, thanks for catching that too! Looks like current version of `reproduce` doesn't use any potentially important options, such as env variables, memory and timeout limits. It should re-use `run_fuzzer` too, let me try that.

Reply:
Uploaded https://github.com/google/oss-fuzz/pull/4324, should work fine, will land on Monday.

Reply:
Landed, the `reproduce` use case should be addressed now as well!

Reply:
Thanks! Seems to work locally and in CI :)